### ISSUE TITLE
infer example

### ISSUE THREAD

**Issue by yassmine-lam** [BinLiang-NLP/Sentic-GCN#4]
Hi,

Could u please share with us how to infer an example using senticgcn_bert model?

Many thanks

**Comment by BinLiang-NLP** [comment]
> Hi,
> 
> Could u please share with us how to infer an example using senticgcn_bert model?
> 
> Many thanks

Hi,

Thanks for your question.
I have added the code of inferring an example for both senticgcn and senticgcn_bert.
Please see the files of [infer.py](https://github.com/BinLiang-NLP/Sentic-GCN/blob/main/infer.py) and [infer_for_bert.py](https://github.com/BinLiang-NLP/Sentic-GCN/blob/main/infer_for_bert.py).

Please let me know if there is any other problem.

Thanks!!!

**Comment by yassmine-lam** [comment]
Hi,

First, thank u for adding these two files. However, I think the model_state_dict_paths is wrong, as the extension of the saved models is not .pkl in state_dict folder.

I tried the infer_for_bert.py code with one of the saved models but I got this error

![Screen Shot 2021-12-30 at 6 57 29 AM](https://user-images.githubusercontent.com/75603019/147725703-3da453b8-ed7d-4b02-b80f-71248f922bdb.png)

Thanks for ur help!
